### ISSUE TITLE
[bug] Version 1.5+ multiple clicks registered

### ISSUE THREAD

**Issue by EricMclaren** [marcocesarato/react-native-input-spinner#38]
Since version 1.5.0, clicking once on the minus or plus button will make the input spinner go straight to the maximum or minimum. If no maximum is specified, the input will continue increasing.

**Comment by marcocesarato** [comment]
Hi, I tried with multiple configuration like:

```js
<InputSpinner
	max={100}
	value={this.state.value}
	onChange={(value) => {
		this.setState({value: value});
	}}
/>
```

```js
<InputSpinner
        min={10}
	max={100}
	value={this.state.value}
	onChange={(value) => {
		this.setState({value: value});
	}}
/>
```

```js
<InputSpinner
        min={10}
	max={100}
/>
```

but I haven't encountered the problem. 
Could it be possible to have an example or more details?

**Comment by marcocesarato** [comment]
Should be fixed on 1.6.0. 
Let me know if this problem is solved for you too.

**Comment by EricMclaren** [comment]
Hi, thanks a lot for your follow up.
I deleted the node_modules and re-installed all dependencies with the new version of the input-spinner. Works Great!
